package/MDAnalysis/analysis/msd.py

```diff
@@ -267,6 +268,10 @@
 class EinsteinMSD(AnalysisBase):
     r"""Class to calculate Mean Squared Displacement by the Einstein relation.
 
+    If `fft=False` so that the "windowed" algorithm is used, the second order
```

Comment:
I think we need to make the parameter an optional addition to the `fft=False` run. MSDs are already expensive (esp without an FFT) and tacking on more compulsory computation is not ideal. Much better to have it as an optional kwarg.

Reply:
Fair enough, that also begs the question on whether this package will move toward cython modules?

Reply:
for the MSD package? There is likely some benefit there. We are generally encouraging of Cython  contributions and have a bunch of extensions already written in Cython. 

If you go down the cython side and need a hand feel free to ping. See setup.py for the minimal setup for a cython extension.